src/Models/Esp32Device.ts

```diff
@@ -42,7 +41,7 @@ export class Esp32Device extends ArduinoDeviceBase {
     let packageRootPath = '';
     let version = '0.0.1';
 
-    if (plat === 'win32') {
+    if (plat === OSPlatform.WIN32) {

```

Comment:
It is better to align the variable name from plat to platform.

Reply:
Fixed.